src/Utils/Composer.php

```diff
@@ -30,8 +30,21 @@ public static function vendorDir(?string $path = null): string
 
     public static function autoloadPath(): string
     {
+        /**
+         * if $_composer_autoload_path is undefined, fall back to vendor/autoload.php in the parent project's directory.
+         *
+         * @link https://getcomposer.org/doc/articles/vendor-binaries.md#finding-the-composer-autoloader-from-a-binary
+         */
         global $_composer_autoload_path;
-        return realpath($_composer_autoload_path) ?: $_composer_autoload_path;
+        if (isset($_composer_autoload_path)) {
+            $autoload_path = $_composer_autoload_path;
+        } elseif (is_dir(codecept_root_dir('vendor'))) {
```

Comment:
I think the two cases that handle the missing global `$_composer_autoload_path` case could be simplified.

```php
$vendorDir = dirname((new \ReflectionClass(Codeception\Codecept::class))->getFilename(), 5);
$autoloadPath = $vendorDir . DIRECTORY_SEPARATOR . 'autoload.php';
```

wp-browser relies on Codeception, which contains the `Codeception\Codecept` class, the main project class.
That is unlikely to move (and not in the current versions) and will always be in the vendor directory (whatever that is and however that is defined) whether wp-browser is a dependency of a project itself or is the project under development.

Reply:
Excellent — I like this much more. I'll push an update in the morning once I've had some sleep :D

Reply:
✅ Done in [29c5612](https://github.com/lucatume/wp-browser/pull/760/commits/29c5612978842de5ffb37b6c627e5041a748c5e1)